packages/x-tailwindcss/src/x-tailwind-plugin/components/suggestion-group/outlined.ts

```diff
@@ -0,0 +1,23 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `outlined` variant for component `suggestion group`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function suggestionGroupOutlined(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+  return {
+    outlined: {
+      backgroundColor: theme('colors.neutral.0'),
+      borderWidth: theme('borderWidth.1'),
+      borderColor: 'var(--suggestion-group-color-50)',
+      '&:hover': {
+        backgroundColor: theme('colors.neutral.10')
+      },
```

Comment:
suggestion: When you add sub-selectors like this one in the middle of more CSS rules, tailwind generates 2 different rules. Moreover, I believe it is easier to read if we group all the styles together before the nested selectors.
![image](https://user-images.githubusercontent.com/68222542/209780401-de8deef0-de2d-4588-9ac7-e23a1cde6c16.png)